From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: SLIGHTLKE <slightlke@outlook.com>
Date: Wed, 21 Oct 2026 15:50:00 +0000
Subject: [PATCH] SIMD MapPalette

This patch was generated by an LLM, with reference to the project Luminol (https://github.com/LuminolCustomArchive/Luminol).

diff --git a/org/bukkit/craftbukkit/map/CraftMapCanvas.java b/org/bukkit/craftbukkit/map/CraftMapCanvas.java
--- a/org/bukkit/craftbukkit/map/CraftMapCanvas.java
+++ b/org/bukkit/craftbukkit/map/CraftMapCanvas.java
@@ -125,7 +125,7 @@
             graphics.dispose();
         }
 
-        byte[] bytes = MapPalette.imageToBytes(temp);
+        byte[] bytes = me.slightlke.isoluminol.optimizations.simd.SIMDMapPalette.imageToBytes(temp);
 
         // Since we now control the size of the image, we can safely use System.arraycopy
         // If x is 0, we can just copy the entire image as width is 128 and height is <=(128-y)
